lib/assets/javascripts/turbograft/remote.coffee

```diff
@@ -99,6 +99,22 @@ class TurboGraft.Remote
         if (radioOrCheck && input.checked) || !radioOrCheck
           callback(input)
 
+  _enabledInputs: (form) ->
+    selector = "input:not([type='reset']):not([type='button']):not([type='submit']):not([type='image']), select, textarea"
+    inputs = Array.prototype.slice.call(form.querySelectorAll(selector))
```

Comment:
Doing this `Array.prototype.slice.call` stuff because form.querySelectorAll does not return an array, but a NodeList. NodeLists lacks some features of arrays like the ability to modify them with concat. This convert it to an array.